ha/core/event_analyzer/parser/parser.py

```diff
@@ -168,8 +170,26 @@ def __init__(self):
         self.cluster_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}cluster_id")
         self.site_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}site_id")
         self.rack_id = Conf.get(const.HA_GLOBAL_INDEX, f"COMMON_CONFIG{_DELIM}rack_id")
+
+        _index = 'cortx'
+        ConfigManager._safe_load(_index, "yaml:///etc/cortx/cluster.conf")
```

Comment:
_safe_load, as per the signature, its a private function. So, it shouldn't be accessible from outside(though it is possible in python). Instead, you can use Conf.load() with your index.(Conf from utils). and then access search API.

Reply:
Also, please use constant for cluster.conf.